inst/ms/aggreCAT.qmd

```diff
@@ -1164,27 +1250,28 @@ aggreCAT::confidence_score_evaluation(
         Brier_Score = "Brier Score",
         Classification_Accuracy = "Classification Accuracy") %>% 
   gt::fmt_number(columns = -c(method, Classification_Accuracy)) %>% 
-  gt::fmt_percent(Classification_Accuracy,scale_values = FALSE,decimals = 0)
+  gt::fmt_percent(Classification_Accuracy,scale_values = FALSE, 
+                  decimals = 0)
 ```
 
 ### Visualising Judgements, Confidence Scores and Forecast Performance
 
-We include two functions for visualising comparison and eavluation of
+We include two functions for visualising comparison and evaluation of
 Confidence Scores across multiple aggregation methods for a suite of
 forecasts from multiple participants,
-[confidence\\_scores\\_ridgeplot]{.fct} and
-[confidencescore\\_heatmap]{.fct}.
-[confidence\\_scores\\_ridgeplot]{.fct} generates ridgeline plots
+[confidence\_scores\_ridgeplot]{.fct} and
+[confidencescore\_heatmap]{.fct}.
+[confidence\_scores\_ridgeplot]{.fct} generates ridgeline plots
 using [ggridges]{.pkg} @ggridges2021 , and displays the distribution of
 predicted outcomes across a suite of forecasts for each aggregation
 method, grouped into separate 'mountain ranges' according to the
-mathematical properties of the aggregation method @fig-ridgeplot.
+mathematical properties of the aggregation method (@fig-ridgeplot).
 
 ```{r}
 #| label: fig-ridgeplot
 #| fig-height: 8
 #| fig-width: 10
-#| fig-cap: "Ridge plots blah blah blah."
+#| fig-cap: "Ridgeline plots illustrating the distribution of aggregated Confidence Scores for the tibble `confidenceSCOREs`, grouped according to mathematical properties of each method."
```

Comment:
Ok, so this is my first time seeing the output of this function and it isn't what I expected. I thought this was supposed to match what @psychtek had done previously where they were grey-ish and had red/blue top/bottom 5% quantiles? The old-school WordArt colour scheme has to go. I get the dashed line is the 0.5 line, but what is the solid line? Distribution mean? Why is it the same for all distirbutions in Weighted Linear Combinations? In terms of overall aesthetics I think we should go back to just standard `theme_bw()` to keep boxes/gridlines (willing to lose gridlines though), and not get rid of the facet label boxes. We should keep things relatively simple and let users "expand" on it with their own ggplot modifications. I would be _very negative_ about this figure if I was reviewing it in a manuscript as it has made aesthetic choices that make the "science" harder to interpret (regardless of my opinions of aesthetics)

Reply:
I can't remember what the previous version of the ridge lines looked like - do we have a image file / and the associated code?

Agree with the principle of not enforcing theme aesthetics on users, and letting users modify instead.

Looking at the source code for the function `confidence_score_ridgeplots()`, the dashed line is the 0.5 probability mark, and the solid line is given by the call to `ggridges::geom_density_ridges_gradient` within the function. I can't figure out what the line represents looking at the ?man pag, but looking at the `ggplot2::labs()` caption call, I can see that the solid line is supposedly the median.

Reply:
- Aaron was working on this. @psychtek is there a PR for this or is it on Dev?